### ISSUE TITLE
[BUG] Reminders module never work for redesigned reminders in iOS14 and macOS11

### ISSUE THREAD

**Issue by Cyberpuncrush** [picklepete/pyicloud#313]
## The problem

Apple Reminders never work. And it seems like only works for old version reminders, not redesigned version in ios14, macos big sur.

## Environment

macOS big sur 11.0.1
Python 3.7
pyicloud0.9.7

- pyiCloud release with the issue (`pip show pyicloud`): 
- Last working pyiCloud release (if known): 
- Service causing this issue: 
- Python version (`python -V`): 
- Operating environment (project deps/Docker/Windows/etc.): 

## Traceback/Error logs

```python
# api auth works and api.refresh works
api.reminders.refresh()
api.reminders.post('Test')

False
```

## Additional information

None

**Comment by RaddedMC** [comment]
Yeah, Apple made some big changes to Reminders with the iOS 13 update that caused users of 12 and lower devices to be forced to either 'upgrade and break' reminders on their old devices or don't upgrade and miss out on the new features to come to reminders. The updated Reminders system is accessible on the website so it might be a matter of updating this API.

**Comment by Cyberpuncrush** [comment]
> The updated Reminders system is accessible on the website

You mean there is a web API way to interact with iOS13 and later reminders.app? If so, where can I find them?

**Comment by RaddedMC** [comment]
> If so, where can I find them?

I can see mine at [icloud.com/reminders](https://icloud.com/reminders). I might be interpreting your question incorrectly but that's the website, I haven't found an API to connect it to Python yet. Notably embeds don't show up but due dates and titles do.

![image](https://user-images.githubusercontent.com/66137579/101353390-7e7c7180-3861-11eb-810d-1c3ae8085950.png)
